includes/HealthChecks.php

```diff
@@ -0,0 +1,332 @@
+<?php
+
+namespace NewfoldLabs\WP\Module\Performance;
+
+use NewfoldLabs\WP\ModuleLoader\Container;
+
+/**
+ * Add performance health checks.
+ */
+class HealthChecks {
+
+	/**
+	 * Dependency injection container.
+	 *
+	 * @var Container
+	 */
+	protected $container;
+
+	/**
+	 * Constructor.
+	 *
+	 * @param Container $container Dependency injection container.
+	 */
+	public function __construct( Container $container ) {
+		$this->container = $container;
+	}
+
+	/**
+	 * Add health checks.
+	 */
+	public function addHealthChecks() {
```

Comment:
I think this works well for now. But if this file grows into a large monolith in the future, do you think it might make sense to create an abstract class for new health checks to inherit from?

For instance, we could have a `HealthChecks` directory with an `AbstractHealthCheck.php` file that defines and documents all the necessary fields. A class like `LinkPrefetchHealthCheck.php` could then inherit from it and include all tests related to link prefetching.

In this file, we can simply call `$manager->addHealthCheck( new LinkPrefetchHealthCheck() );`. This approach would provide better validation and typing in PHP, prevent files from becoming too large, group related functionality, and avoid people adding random data directly into the manager. What do you think?